File: cpp/oppgave3.cpp
#include <iostream>

using namespace std;

int main()
{

int numbers[] = {1, -1, 8, -5};
int n = sizeof(numbers)/ sizeof(numbers[0]);

int min = INT_MAX;

for(int i=0; i<n; i++){
    if(numbers[i]<min)
    min = numbers[i];
}

cout<<min;

}

File: go/oppgave3.go
package main

import (
	"fmt"
)

func minFind(numbers []int) int {
	min := numbers[0]
	for _, num1 := range numbers {
		if num1 < min {
			min = num1
		}
	}
	return min
}

func main() {
	numbers := []int{1, -1, 8, -5}
	min := minFind(numbers)
	fmt.Printf("%d", min)
}
